common/PrerenderData.jsm

```diff
@@ -89,14 +89,14 @@ this.PrerenderData = new _PrerenderData({
       icon: "pocket",
       id: "topstories",
       order: 1,
-      title: {id: "header_recommended_by", values: {provider: "Pocket"}}
+      title: {id: "header_recommended_by_upper", values: {provider: "POCKET"}}
```

Comment:
You're would also need to change this in the prefs config of `Activity-Stream.jsm` (see `feeds.section.topstories.options`) but I'm not sure that's the right thing to do given that in some languages the translation won't be all uppercase?

Reply:
possibly just hardcoding pocket into the string is a reasonable option too since there will be no other providers?

Thoughts @Mardak ?

Reply:
For reference:

<img width="394" alt="screen shot 2018-07-25 at 5 41 55 pm" src="https://user-images.githubusercontent.com/36629/43229232-198a2bca-9032-11e8-8641-4f99a5126f71.png">